backend/migrations/versions/5a798acc6282_create_version_mf.py

```diff
@@ -0,0 +1,101 @@
+"""create_version_mf
+
+Revision ID: 5a798acc6282
+Revises: 075d344ae2cc
+Create Date: 2024-10-10 17:25:09.687099
+
+"""
+
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy import orm
+
+from dataall.modules.metadata_forms.db.metadata_form_models import (
+    MetadataForm,
+    MetadataFormVersion,
+    AttachedMetadataForm,
+    MetadataFormField,
+)
+
+# revision identifiers, used by Alembic.
+revision = '5a798acc6282'
+down_revision = '075d344ae2cc'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    bind = op.get_bind()
+    session = orm.Session(bind=bind)
+
+    op.create_table(
+        'metadata_form_version',
+        sa.Column('metadataFormUri', sa.String(), nullable=False),
+        sa.Column('version', sa.Integer(), nullable=False),
+        sa.ForeignKeyConstraint(
+            ['metadataFormUri'], ['metadata_form.uri'], name='f_key_version_metadata', ondelete='CASCADE'
+        ),
+        sa.PrimaryKeyConstraint('metadataFormUri', 'version'),
+    )
+    for mf in session.query(MetadataForm).all():
+        version = MetadataFormVersion(metadataFormUri=mf.uri, version=1)
+        session.add(version)
+        session.commit()
+
+    op.add_column('attached_metadata_form', sa.Column('version', sa.Integer()))
+    for amf in session.query(AttachedMetadataForm).all():
+        amf.version = 1
+        session.commit()
+    op.alter_column('attached_metadata_form', 'version', nullable=False)
```

Comment:
nit: just a callout I think may be able to do same with `server_default` parameter such as 
```
op.add_column('attached_metadata_form', sa.Column('version', sa.Integer(), server_default=1))
```

But changing nullable value after would think works as well

Reply:
I don't want the column to have the default value, I want it to be explicitly filled. Otherwise we can have troubles with debugging.